### PR TITLE
Release v0.10.4 -- L7+8 pair fixes

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

**Comment by jhkennedy** [comment]
Golden test jobs all finished successfully, so I think this is ready for review.

I am still looking through the outputs, and will merge once I get an approval and am satisfied wout the outputs.

**Comment by jhkennedy** [comment]
Everything looks good in the golden tests. L8 + L7 pair is significantly improved.

main:
LC08_L1TP_060018_20130330_20200912_02_T1_X_LE07_L1TP_061018_20130415_20200907_02_T1_G0120V02_P008_IL_ASF_OD

develop:
LC08_L1TP_060018_20130330_20200912_02_T1_X_LE07_L1TP_061018_20130415_20200907_02_T1_G0120V02_P023_IL_ASF_OD

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**
